support-frontend/assets/components/subscriptionCheckouts/threeTierTerms.tsx

```diff
@@ -20,46 +26,106 @@ const threeTierTerms = css`
 	}
 `;
 
+export const productNameUSTier3 = 'Guardian Weekly and All-Access Digital';
+
 const termsLink = (linkText: string, url: string) => (
 	<a target="_blank" rel="noopener noreferrer" href={url}>
 		{linkText}
 	</a>
 );
 
-export default function ThreeTierTerms(props: {
+export default function Tier3Terms(props: {
 	paymentFrequency: 'month' | 'year';
 	paymentMethod?: PaymentMethod;
+	countryGroupId?: CountryGroupId;
 }): JSX.Element {
-	const threeTierTermsLink =
-		'https://www.theguardian.com/info/article/2024/jul/19/digital-print-terms-and-conditions';
 	const paymentFrequencyName =
 		props.paymentFrequency === 'year' ? 'annual' : 'monthly';
+	const productName =
+		props.countryGroupId === 'UnitedStates'
+			? productNameUSTier3
```

Comment:
The convention i the codebase it to name things as you say them e.g. `TierThreeUSProductName`. Don't mind, but as previously stated keeping with convention makes guessing easier down the line.

Reply:
Ok, will try to keep with convention, changed